### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by solo-bog**
installing packages, linters and linting

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    rules: {
+        'linebreak-style': 'off',
+        'prettier/prettier': [
+            'error',
+            {
+                endOfLine: 'auto',
+            },
+        ],
+    },
+};
```
Comment (MaxKalashnyk) [inline .eslintrc.js]: @solo-bog, you can configure your IDE for adding additional line in the end of each file

File: src/serviceWorker.ts [hunk 2]
```diff
@@ -11,9 +11,9 @@
 // opt-in, read https://bit.ly/CRA-PWA
```
Comment (MaxKalashnyk) [inline src/serviceWorker.ts]: We can get rid of this file

File: package.json [hunk 3]
```diff
@@ -12,10 +12,14 @@
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
     "@types/react-redux": "^7.1.7",
+    "bootstrap": "5.0.2",
     "react": "^17.0.2",
+    "react-bootstrap": "^1.6.1",
```
Comment (MaxKalashnyk) [inline package.json:17]: Do you really need this package in this project?

**Review by MaxKalashnyk — COMMENTED** [review]
LGTM! Just left minor comments.

File: package.json (hunk 3 above)
Comment (solo-bog) [reply package.json:17]: some next commits use bootstrap components. In last commits I removed these components and then removed this package

**Review by MaxKalashnyk — APPROVED** [review]
Good job!

**Pull request merged**